services/datamanager/internal/data_manager_test_helper.go

```diff
@@ -1,25 +0,0 @@
-// Package internal implements a data manager service definition with additional exported functions for
```

Comment:
this interface no longer needs to exists. I've architected the builtin data manger, sync and capture packages to allow us to inject the dependencies we need to which we previously accomplished with this interface.